packages/dev/docs/pages/Support.mdx

```diff
@@ -12,10 +12,56 @@ export default Layout;
 
 ---
 category: Foundation
-description: How to contribute features, report bugs, or make suggestions to improve React Spectrum.
+description: How to get help with React Spectrum, how to report issues, and our development roadmap.
 keywords: [react spectrum support, react spectrum help]
 ---
 
 # Support
 
-For feedback, bug reports, general questions about React Spectrum, please visit https://github.com/adobe/react-spectrum/issues/.
+This page includes information about how to get help with React Spectrum, how to report issues, and our development roadmap.
```

Comment:
This page includes getting help with React Spectrum, reporting issues, and information on our development roadmap.

Reply:
"This page includes getting help" doesn't make sense to me. Feels like it needs the "information about"